components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8061,7 +8061,7 @@ components:
             - PRIVATE
             - RESTRICTED
           x-otherScopes:
-            - apim:api_publish
+            - apim:api_publishAPI:
```

Comment:
```suggestion
            - apim:api_publish
```

Reply:
@akshitha10110 please regenerate the publisher code and commit as well as this was a change to the OAS definition